news/2025/2025-07-20-4-digit-world-cup-2025-concludes.md

```diff
@@ -0,0 +1,252 @@
+
+In the blink of an eye, the momentum was now squarely on the side of France, who had rattled off 4 straight points and now had the opportunity to use their fourth pick to reach match point and potentially hand the United States their first loss in three years. They selected [HD3](https://osu.ppy.sh/beatmapsets/2373888#osu/5126280), which on paper was a great pick, being the final tech map in a matchup where they had won the other two. But the United States snapped right back, with a ::{ flag=US }:: [LUNAISTABBY](https://osu.ppy.sh/users/4673649) FC and high supporting scores granting them a break point with a wide margin of 959,711. Now at 4–4, they would reach match point by picking the anti-mod [FM2](https://osu.ppy.sh/beatmapsets/2210416#osu/5132658) and dominating France with another nearly 1 million point difference.
+
+It was now France's final pick. With pressure at an all-time high, they decided to stake their winner's bracket lives on [HR2](https://osu.ppy.sh/beatmapsets/1262817#osu/2639476), a hybrid map of precision and flow aim alt. The map was close until the very end, with ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990) and ::{ flag=FR }:: [Draxarta](https://osu.ppy.sh/users/11129041) matching FCs, supported by ::{ flag=US }:: [Uzumaki](https://osu.ppy.sh/users/13310147) and ::{ flag=FR }:: [fiaee](https://osu.ppy.sh/users/10325072) matching chokes, but ultimately the supporting scores for the Americans were better as they finished off the French team on a nail-biter 78,969 victory for a final break point.
+
+Both teams played incredibly well this match, with the United States team achieving the top score of the weekend on 5/6 of their points, and the French team doing so on 3/4 of theirs. Besides the final map, most picks in this match were won in dominant fashion, with only 2 other picks being decided by a margin of 450,000 or below, and many being close to 1 million; an extremely rare statistic for a match that finished at 6–4.
+
+### Semifinals: ::{ flag=DE }:: Germany vs. ::{ flag=ID }:: Indonesia
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/o3qqffDMWZg" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Moving into the Semifinals weekend, the 2nd seed Germany, after their unexpected loss against Sweden in the previous week, had to face off against the 17th seed Indonesian roster to avoid being eliminated.
```

Comment:
```suggestion
Moving into the Semifinals weekend, the 2nd-seeded Germany, after their unexpected loss against Sweden in the previous week, had to face off against the 17th-seeded Indonesian roster to avoid being eliminated.
```